website/docs/docs/user-interface/settings/preferences.md

```diff
@@ -2,49 +2,48 @@
 title: Preferences
 ---
 
-This screen specifies the PactFlow Preferences panels and what types of settings you can change.
+In this screen, you can see which settings you can modify in PactFlow Preferences panels. There are two types of preferences:
 
 ## Personal preferences
 
 These are the settings that affect individual users.
 
 #### Show welcome dialog
 
-This setting controls whether the "Welcome to PactFlow" dialog is shown to the user when they login.
+When a user logs in to PactFlow, this setting controls whether the "Welcome to PactFlow" dialog appears.
 
 #### Show an alert when my tokens are about to expire
 
-This setting controls whether the token expiry alert is shown to the user on the main dashboard when any of their
-API tokens have expired or are within seven days of expiring. This will only occurr if token expiration is enabled
-by an administrator in the System preferences.
+This setting determines if the user sees the token expiry alert on the main dashboard for any API tokens that are either expired or expiring within seven days. An administrator must enable token expiration in the System preferences for this to happen.
 
 ## System preferences
 
-Theses are global system settings and can only be changed by an administrator.
+These are global system settings and can only be changed by an administrator.
 
 #### API Token expiration
 
-By default, user and system account tokens don't expire, but can be re-generated at any time to invalidate the old tokens 
-and create new ones. This setting enables the tokens to automatically expire after a fixed period of time, which can be
-specified in units of hours, days, weeks or months.
+By default, user and system account tokens don't expire, but can be re-generated at any time to invalidate the old tokens and create new ones. This setting enables tokens to automatically expire after a fixed period of time, which can be specified in units of hours, days, weeks or months.
 
-Once a token has expired, a new one will need to be generated from the [API Tokens setting page](./api-tokens) (for users) or the [user
-management screen](./users#system-accounts) (for system accounts).
+After a token expires, the users can generate a new one from the [API Tokens setting page](./api-tokens), while the system account holders can do so from the [user
```

Comment:
System accounts holders doesn't really make sense as a term, as there is no "holder" behind them (they are for machines).


```suggestion
After a token expires, the users can generate a new one from the [API Tokens setting page](./api-tokens), while users with the appropriate `system_account:manage` permission can regenerate system account tokens from the [user
```